sql/core/src/test/resources/sql-tests/inputs/interval.sql

```diff
@@ -0,0 +1,40 @@
+--
```

Comment:
the file created as a suggestion from https://github.com/apache/spark/pull/26410#discussion_r343126470